crates/forge/tests/it/zk/deploy.rs

```diff
@@ -24,8 +24,24 @@ forgetest_async!(multiple_deployments_of_the_same_contract, |prj, cmd| {
     );
 });
 
+forgetest_async!(can_deploy_via_create2, |prj, cmd| {
+    setup_deploy_prj(&mut prj);
+    run_zk_script_test(
+        prj.root(),
+        &mut cmd,
+        "./script/Create2.s.sol",
+        "Create2Script",
+        None,
+        3,
+        Some(&["-vvvvv"]),
+    );
+});
+
 fn setup_deploy_prj(prj: &mut TestProject) {
```

Comment:
I appreciate the separate test, but I'd also like the environment to be separate, so `deploy_prj` is only Deploy & Greeter, meanwhile this one can have it's own "project".
In the end, it's still 2 tests run separately in parallel, so the gain would be miniscule, so in reality is for good practice and keeping the section clean